### PR TITLE
Change edit_uri default branch to v0.x.x in mkdocs.yml

### PR DESCRIPTION
**Pull request by cwasicki**

### CODE REVIEW

**Comment by cwasicki** [comment]
Looks like I missed something, will have a look later.

**Review by llucax — DISMISSED** [review]

File: cookiecutter/{{cookiecutter.github_repo_name}}/mkdocs.yml [hunk 1]
```diff
@@ -8,9 +8,7 @@ site_author: "{{cookiecutter.author_name}}"
 copyright: "Copyright © {{copyright_year}} {{cookiecutter.author_name}}"
 repo_name: "{{cookiecutter.github_repo_name}}"
 repo_url: "https://github.com/{{cookiecutter.github_org}}/{{cookiecutter.github_repo_name}}"
-# TODO(cookiecutter): "main" is the GitHub repo default branch, you might want to update it
-# if the project uses a different default branch.
-edit_uri: "edit/main/docs/"
+edit_uri: "edit/v0.x.x/docs/"
```
Comment (llucax) [inline cookiecutter/{{cookiecutter.github_repo_name}}/mkdocs.yml:13]:
```suggestion
# TODO(cookiecutter): "v0.x.x" is the GitHub repo default branch, you might want to update it
# if the project uses a different default branch.
edit_uri: "edit/v0.x.x/docs/"
```

**Review by llucax — COMMENTED** [review]
Oh, the tests are actually running cookiecutter and checking the output, and the TODOs are listed there, this is why is failing. Did you use the command to [update the golden tests](https://github.com/frequenz-floss/frequenz-repo-config-python/blob/v0.x.x/CONTRIBUTING.md#golden-tests) or did you update them manually? I'm not sure if that's handled by that command.

In any case, I think the TODO should stay, maybe even more now than before, as `0.x.x` is not very common at all as default branch name.

**Comment by llucax** [comment]
BTW, if this is too distracting for you, I can finish the PR myself, thanks for going straight to a PR instead of an issue!

**Comment by cwasicki** [comment]
> Did you use the command to [update the golden tests](https://github.com/frequenz-floss/frequenz-repo-config-python/blob/v0.x.x/CONTRIBUTING.md#golden-tests) or did you update them manually?

I did it manually.

> BTW, if this is too distracting for you, I can finish the PR myself, thanks for going straight to a PR instead of an issue!

If you can quickly fix it would be highly appreciated :)

**Comment by llucax** [comment]
Force-merging, as I only re-generated the golden tests (and applied my own suggestion :smile: )

**Pull request merged**